observatory-platform/observatory/platform/utils/file_utils.py

```diff
@@ -333,3 +333,45 @@ def zip_files(file_list: List[str]):
                 shutil.copyfileobj(f_in, f_out)
 
     return zip_list
+
+
+def unzip_files(*, file_list: List[str], output_dir: str = None):
```

Comment:
Perhaps give this function a name that communicates that it is unzipping gzip files, e.g. gunzip_files